doc/whats_new.rst

```diff
@@ -19,6 +19,8 @@ Changelog
     - Add support for running :class:`sklearn.model_selection.GridSearchCV` on :class:`pyglmnet.GLM` objects by `Mainak Jas`_.
     - Improved testing of gradients for optimization by `Mainak Jas`_ and `Pavan Ramkumar`_.
     - Improved testing of estimated coefficients at convergence by `Mainak Jas`_ and `Pavan Ramkumar`_
+    - Made temporary file handling OS independent and usable on a cluster `Scott Otterson`_.
+    - Misc. documentation updates `Scott Otterson`_.
```

Comment:
The what's new page is only for changes that users are affected by in terms of API / bug fixes etc. Don't think this line qualifies for that.

Reply:
Documentation is user visible, and the examples and certain tests didn't run on Windows.  they would also cause hard-to-find errors on a cluster (depending upon the cluster).  But it doesn't matter to me if you delete those lines.  Just trying to understand...

Reply:
yes the OS independent thing you already mentioned in the line before. Basically, this page is used when making a new release to inform users what they should be looking out for. So part of the job is to of course give credit where it is due and part of it is to be as informative as possible when making release notes. That's why we normally leave out changes to examples etc.